client/store/redux/orders.js

```diff
@@ -0,0 +1,52 @@
+import axios from 'axios';
+
+const SET_ALLORDERS = 'SET_ALLORDERS';
+
+export const setAllOrders = (orders) => {
+  return {
+    type: SET_ALLORDERS,
+    orders,
+  };
+};
+
+export const fetchAllOrders = () => {
+  try {
```

Comment:
You are not `try` `catch`ing the function you are creating as a `thunk`, you are trying to catch the exception from the `await`.